packages/core/app-layout/src/components/aboutSection/AppAboutSection.cy.ts

```diff
@@ -100,6 +102,47 @@ describe('<AppAboutSection />', () => {
     cy.getTestId('about-divider-section').should('contain.text', dividerText)
   })
 
+  it('should correctly render title in both props and slots', () => {
+    const titleProps = 'Cats are Cool'
+    const titleSlot = 'Cats are Awesome'
+
+    // First mount with prop title
+    cy.mount(AppAboutSection, {
+      props: {
+        title: titleProps,
+      },
+    })
+
+    cy.get('.kong-ui-app-about-section').should('exist')
+    cy.getTestId('about-section-title').should('be.visible')
+    cy.getTestId('about-section-title').should('contain.text', titleProps)
+
+    // Now mount with slot title
+    cy.mount(AppAboutSection, {
```

Comment:
Can you split this into separate tests rather than mounting the component twice?